pkg/registry/authentication/selfsubjectreview/rest.go

```diff
@@ -93,3 +93,9 @@ func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 
 	return selfSAR, nil
 }
+
+var _ rest.SingularNameProvider = &REST{}
+
+func (r *REST) GetSingularName() string {
+	return "selfsubjectrulesreview"
```

Comment:
@ardaguclu Hello, it seems like a bug, but I don't know why the tests are passing. I will fix this on promoting selfsubjectreviews to beta, jfyi.

Reply:
The name is `selfsubjectreview`.